festim/exports/derived_quantities/derived_quantities.py

```diff
@@ -105,6 +109,7 @@ def filename(self, value):
     def make_header(self):
         header = ["t(s)"]
         for quantity in self:
+            quantity.show_units = self.show_units
```

Comment:
Should we add a deprecation warning here if `self.show_units` is `False`? Warning the users that the new behaviour in a couple of releases will be to show units